pkgs/desktops/xfce/default.nix

```diff
@@ -81,6 +81,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   xfce4-terminal = callPackage ./applications/xfce4-terminal { };
 
+  xfce4-screensaver = callPackage ./applications/xfce4-screensaver { };
```

Comment:
Wasn't sure if it might make sense to make this optional. It feels like a pretty integral part of the Xfce desktop environment to me (certainly more so, than, e.g., mousepad or parole), but the fact that no one had bothered to package it yet would seem to indicate that might not be a universal view.

`mate.nix` provides `environment.mate.excludePackages`, which allows any of the non-essential packages to be excluded without requiring a separate option for each one seems like it could be a nice solution here, but also seems outside the scope of this pull request.